### PR TITLE
Docs: Remove UNIQUE keyword as it is not supported in Flink

### PR DESCRIPTION
**Pull request by PrabhuJoseph**
Remove UNIQUE keyword as it is not supported in Flink.

### CODE REVIEW

**Comment by PrabhuJoseph** [comment]
@pvary Please review this when you have time. Thanks.

**Comment by aokolnychyi** [comment]
cc @pvary @stevenzwu

File: docs/flink-writes.md [hunk 1]
```diff
@@ -69,7 +69,7 @@ Iceberg supports `UPSERT` based on the primary key when writing data into v2 tab
 
 ```sql
 CREATE TABLE `hive_catalog`.`default`.`sample` (
-    `id`  INT UNIQUE COMMENT 'unique id',
+    `id`  INT COMMENT 'unique id',
```
Comment (pvary) [inline docs/flink-writes.md:72]: nit: Could we remove the extra space after the column name too?

Comment (PrabhuJoseph) [reply docs/flink-writes.md]: Thanks for reviewing. Have removed the extra space.

**Review by pvary — APPROVED** [review]

**Comment by pvary** [comment]
Thanks for the fix @PrabhuJoseph!

**Pull request merged**